### PR TITLE
Update roadmap

### PR DESCRIPTION
**Pull request by trang-erskine**
Update roadmap with the latest information

### CODE REVIEW

**Comment by calvin-lau-sig7** [comment]
Generally looks good to me too!

I can do this as a further PR, we usually refer to other pages on the website with 'Sentence case' style. So something like:
> Release '[Hide this page](https://github.com/alphagov/govuk-design-system-backlog/issues/213)' component

Also, perhaps it might read nicer to add lead-in sentences to the bullet lists (just under the headings)?

### Recently shipped
We recently:

### Working on now
We're working to:

### Coming up next
We're getting ready to:

### Future plans
We're planning to:

File: src/community/roadmap/index.md.njk [hunk 1]
```diff
@@ -12,32 +12,37 @@ Some things on the roadmap might change — the purpose is to tell you what’s
 
 See our [Github team board](https://github.com/orgs/alphagov/projects/4) for more details on our plans and day-to-day activities.
 
-Last updated 27 September 2021.
+Last updated 28 February 2022.
 
 ## Recently shipped
```
Comment (calvin-lau-sig7) [inline src/community/roadmap/index.md.njk:18]:
```suggestion

We recently:
```

File: src/community/roadmap/index.md.njk [hunk 2]
```diff
@@ -12,36 +12,46 @@ Some things on the roadmap might change — the purpose is to tell you what’s
...
-- Completed a mini discovery on how different teams are prototyping across government
-- Released [GOV.UK Frontend v.3.14.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.14.0)
-- Revised accordion design contributed by GOV.UK to address an accessibility issue
-- Released [GOV.UK Frontend Version 4.0.0](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0), includes revised accordion, fixes to headers and footers and a number of breaking changes
+We recently:
+
+- released [GOV.UK Frontend v3.13.1](https://github.com/alphagov/govuk-frontend/releases/tag/v3.13.1)
+- published [accessibility acceptance criteria for contributions](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md)
+- completed a mini discovery on how different teams are prototyping across government
+- released [GOV.UK Frontend v.3.14.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.14.0)
```
Comment (lfdebrux) [inline src/community/roadmap/index.md.njk:24]:
```suggestion
- released [GOV.UK Frontend v3.14.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.14.0)
```

File: src/community/roadmap/index.md.njk [hunk 3]
```diff
@@ -12,36 +12,46 @@ Some things on the roadmap might change — the purpose is to tell you what’s
...
+- revised the accordion design contributed by GOV.UK to address an accessibility issue
+- released [GOV.UK Frontend Version 4.0.0](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0), includes the revised 'Accordion' component, fixes to headers and footers and a number of breaking changes
```
Comment (lfdebrux) [inline src/community/roadmap/index.md.njk:26]:
```suggestion
- released [GOV.UK Frontend v4.0.0](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0), includes the revised 'Accordion' component, fixes to headers and footers and a number of breaking changes
```

File: src/community/roadmap/index.md.njk [hunk 4]
```diff
@@ -12,36 +12,46 @@ Some things on the roadmap might change — the purpose is to tell you what’s
...
+- improve how [users can find out what we are working on](https://github.com/alphagov/govuk-design-system/issues/1410)
+- released [GOV.UK Frontend v.4.0.1](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.1)
```
Comment (lfdebrux) [inline src/community/roadmap/index.md.njk:28]:
```suggestion
- released [GOV.UK Frontend v4.0.1](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.1)
```

**Review by lfdebrux — COMMENTED** [review]
Version numbers need a little clean up

**Review by lfdebrux — APPROVED** [review]

File: src/community/roadmap/index.md.njk [hunk 5]
```diff
@@ -12,36 +12,46 @@ Some things on the roadmap might change — the purpose is to tell you what’s
 
 See our [Github team board](https://github.com/orgs/alphagov/projects/4) for more details on our plans and day-to-day activities.
 
-Last updated 27 September 2021.
+Last updated 28 February 2022.
```
Comment (calvin-lau-sig7) [inline src/community/roadmap/index.md.njk:15]:
```suggestion
Last updated 10 March 2022.
```

File: src/community/roadmap/index.md.njk [hunk 6]
```diff
@@ -12,36 +12,46 @@ Some things on the roadmap might change — the purpose is to tell you what’s
...
 
-- Host community co-design work to [turn task-list into a component](/patterns/task-list-pages/)
-- Host community co-design work to make maps better
-- Improve how [users can find out what we are working on](https://github.com/alphagov/govuk-design-system/issues/1410)
+We're working to:
+- host community co-design work to [turn task-list into a component](/patterns/task-list-pages/)
+- host community co-design work to make maps better
+- release the [Hide this page](https://github.com/alphagov/govuk-design-system-backlog/issues/213) component
+- release the help users to [prove who they are with a text message](https://github.com/alphagov/govuk-design-system-backlog/issues/239) pattern
+- review changes to [Start page](https://github.com/alphagov/design-system-team-internal/issues/552) pattern
```
Comment (calvin-lau-sig7) [inline src/community/roadmap/index.md.njk:38]:
```suggestion
```

File: src/community/roadmap/index.md.njk [hunk 7]
```diff
@@ -12,36 +12,46 @@ Some things on the roadmap might change — the purpose is to tell you what’s
...
+- published [accessibility acceptance criteria for contributions](https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/test-components-using-accessibility-acceptance-criteria.md)
+- completed a mini discovery on how different teams are prototyping across government
+- released [GOV.UK Frontend v3.14.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.14.0)
+- revised the accordion design contributed by GOV.UK to address an accessibility issue
+- released [GOV.UK Frontend v4.0.0](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.0), includes the revised 'Accordion' component, fixes to headers and footers and a number of breaking changes
+- improve how [users can find out what we are working on](https://github.com/alphagov/govuk-design-system/issues/1410)
+- released [GOV.UK Frontend v4.0.1](https://github.com/alphagov/govuk-frontend/releases/tag/v4.0.1)
+- updated the [Accessible autocomplete](https://github.com/alphagov/accessible-autocomplete/releases/tag/v2.0.4) component and clarified [future plan](https://github.com/alphagov/accessible-autocomplete/issues/532)
```
Comment (calvin-lau-sig7) [inline src/community/roadmap/index.md.njk:29]:
```suggestion
- updated the [Accessible autocomplete](https://github.com/alphagov/accessible-autocomplete/releases/tag/v2.0.4) component and clarified [future plan](https://github.com/alphagov/accessible-autocomplete/issues/532)
- made changes to [Start page](https://github.com/alphagov/design-system-team-internal/issues/552) pattern
```

**Comment by lfdebrux** [comment]
@calvin-lau-sig7 updates looked good to me, I committed them. Are we ready to merge this PR?

**Pull request merged**